### ISSUE TITLE
Highlights not saved

### ISSUE THREAD

**Issue by quarterwave** [prateek-chawla/Highlighter-Notes-Browser-Extension#1]
Highlights are no longer saved in Firefox 91.5.0esr (64-bit). Even highlights made in previous Firefox versions, where it used to be saved, are no longer displayed.

**Comment by quarterwave** [comment]
@nodiscc @prateek-chawla Yes, it's the same error (Firefox 91.5.0esr 64-bit, plugin version 1.0.0):
```
Error: QuotaExceededError: storage.sync API call exceeded its quota limitations.
```

**Comment by nodiscc** [comment]
> modify the extension to use local storage instead

This would be great :+1: , without the ability to persist/save highlight, the extensions is a lot less useful

> comes with the down side that the highlights won't be synced across different browser instances.

An option to sync/import/export highlights to/from a file would be a nice alternative (this way highlights could be "synchronized" using a standard file synchronization solution, or even external storage). I can open a separate issue for this later. uBlock origin is able to load/save its settings to a file so it could be a useful reference.

localstorage can also be exported manually, from the console [[1]](https://stackoverflow.com/questions/13335967/export-data-in-localstorage-for-later-re-import)

```js
JSON.stringify(localStorage)
```
